src/app/backend/replicasetdetail.go

```diff
@@ -157,6 +163,24 @@ func DeleteReplicaSetWithPods(client *client.Client, namespace string, name stri
 	return nil
 }
 
+// Updates number of replicas in Replica Set based on Replica Set Spec
+func UpdateReplicasCount(client *client.Client, namespace string, name string,
```

Comment:
Use `client client.Interface` and then pkg/client/unversioned/testclient to be able to test this.